front-end-node/NodeInspectorOverrides.js

```diff
@@ -134,3 +150,47 @@ function getAllUiSourceCodes() {
     return p.name();
   }
 }
+
+var notifications = [];
+
+function processDebuggerPaused(event) {
```

Comment:
Wow, I was assuming it was loaded with isolation... I'll do the second solution. ;)